### PR TITLE
Pipe follower demo. Motala

### PR DESCRIPTION
**Pull request by ignaciotb**

### CODE REVIEW

File: smarc_bringup/launch/auv_scenarios.launch [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <launch>
 
 	<!-- Gazebo environment -->
-	<arg name="gazebo_env" default="random_rocks"/>
+	<arg name="gazebo_env" default="pipe_following"/>
```
Comment (nilsbore) [inline smarc_bringup/launch/auv_scenarios.launch:4]: This should be part of a separate launch file that calls this. Like `pipe_following_demo.launch`.

File: smarc_bringup/launch/auv_system.launch [hunk 2]
```diff
@@ -7,13 +7,13 @@
 	<!-- Use the navigation stack or the GT from Gazebo -->
 	<arg name="navigation_on" default="true"/>
 	<!-- Initial pose of the AUV -->
-	<arg name="x" default="30"/>
-	<arg name="y" default="0"/>
-	<arg name="z" default="-90"/>
+	<arg name="x" default="244.3"/>
+	<arg name="y" default="-22.4"/>
+	<arg name="z" default="-91.0"/>
+	<arg name="yaw" default="3.0"/>
```
Comment (nilsbore) [inline smarc_bringup/launch/auv_system.launch:13]: Same thing here. This should be part of a separate launch file that calls this. Like `pipe_following_demo.launch`.

File: smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
```
Comment (nilsbore) [inline smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py:1]: The above binary file should not be added to the repo.

Comment (nilsbore) [reply smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py:1]: I.e. `smarc_keyboard_teleop/scripts/.lolo_auv_teleop.py.swp`

File: smarc_lm_visualizer/src/landmark_visualizer_node.cpp [hunk 4]
```diff
@@ -135,6 +135,7 @@ class LandmarkVisualizer{
             markers.color.r = 0.0;
             markers.color.g = 1.0;
             markers.color.b = 0.0;
+            markers.mesh_resource = "package://smarc_worlds/world_models/large_rock/meshes/large_rock.dae";
```
Comment (nilsbore) [inline smarc_lm_visualizer/src/landmark_visualizer_node.cpp:138]: Should probably be a parameter?

File: smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py (hunk 3 above)
Comment (ignaciotb) [reply smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py:1]: I don't even know where that came from

Comment (ignaciotb) [reply smarc_keyboard_teleop/scripts/lolo_auv_teleop_joystick.py:1]: Removed

**Pull request closed without merging**